judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ServicesController.java

```diff
@@ -0,0 +1,29 @@
+package dev.hltech.dredd.interfaces.rest.contracts;
+
+import dev.hltech.dredd.domain.contracts.ServiceContractsRepository;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.AllArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class ServicesController {
```

Comment:
After our discussion I would not move this endpoint to separate controller nor change its path until we have a clear vision what we are trying to achieve.